test.js

```diff
@@ -205,6 +222,172 @@ test('`onEviction` option method is called after `maxSize` is exceeded', t => {
 	t.true(isCalled);
 });
 
+test('max age - should remove the item on get it again', async t => {
+	const lru = new QuickLRU({maxSize: 10, maxAge: 90});
+	lru.set('1', 'test');
+	await sleep(200);
+	t.is(lru.get('1'), undefined);
+});
+
+test('max age - a non recent item can also expire', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', 'test');
+	lru.set('2', 'test2');
+	lru.set('3', 'test4');
+	await sleep(200);
+	t.is(lru.get('1'), undefined);
+});
+
+test('max age - set the item again should refresh the expiration time', async t => {
+	const lru = new QuickLRU({maxSize: 2, maxAge: 100});
+	lru.set('1', 'test');
+	await sleep(50);
+	lru.set('1', 'test2');
+	await sleep(50);
+	t.is(lru.get('1'), 'test2');
+});
+
+test('max age - once an item expires the eviction function should be called', async t => {
+	t.timeout(1000);
+	const expectKey = '1';
+	const expectValue = 'test';
+
+	let isCalled = false;
+	let actualKey;
+	let actualValue;
+	const onEviction = (key, value) => {
+		isCalled = true;
+		actualKey = key;
+		actualValue = value;
+	};
+
+	const lru = new QuickLRU({
+		maxSize: 2,
+		maxAge: 100,
+		onEviction
+	});
+
+	lru.set(expectKey, expectValue);
+
+	await sleep(200);
+
+	t.is(lru.get('1'), undefined);
+	t.true(isCalled);
+	t.is(actualKey, expectKey);
+	t.is(actualValue, expectValue);
+});
+
+test('max age - once an non recent item expires the eviction function should be called', async t => {
+	t.timeout(1000);
+	const expectKeys = ['1', '2'];
+	const expectValues = ['test', 'test2'];
+
+	let isCalled = false;
+	const actualKeys = [];
+	const actualValues = [];
+	const onEviction = (key, value) => {
+		isCalled = true;
+		actualKeys.push(key);
+		actualValues.push(value);
+	};
+
+	const lru = new QuickLRU({
+		maxSize: 2,
+		maxAge: 100,
+		onEviction
+	});
+
+	lru.set('1', 'test');
+	lru.set('2', 'test2');
+	lru.set('3', 'test3');
+	lru.set('4', 'test4');
+	lru.set('5', 'test5');
+
+	await sleep(200);
+
+	t.is(lru.get('1'), undefined);
+	t.true(isCalled);
+	t.deepEqual(actualKeys, expectKeys);
+	t.deepEqual(actualValues, expectValues);
+});
+
+test('max age - on recise max aged items should also be evicted', async t => {
```

Comment:
```suggestion
test('max age - on resize, max aged items should also be evicted', async t => {
```

I think you could have spent a little more time trying to reduce typos.